### PR TITLE
binance: update apis

### PR DESCRIPTION
**Pull request by sc0Vu**
In this PR, I updated apis in binance.

Changes:
* remove deprecated apis
* add more papis

### CODE REVIEW

File: ts/src/binance.ts [hunk 1]
```diff
@@ -556,7 +556,6 @@ export default class binance extends Exchange {
                         'ticker/price': { 'cost': 1, 'noSymbol': 2 },
                         'ticker/bookTicker': { 'cost': 1, 'noSymbol': 2 },
                         'openInterest': 1,
-                        'pmExchangeInfo': 1,
```
Comment (carlosmiei) [inline ts/src/binance.ts:559]: @sc0Vu where did you see that this endpoint was deprecated? can't find it

Comment (sc0Vu) [reply ts/src/binance.ts:559]: @carlosmiei 
fapi: https://binance-docs.github.io/apidocs/futures/en/#change-log
dapi: https://binance-docs.github.io/apidocs/delivery/en/#change-log

Although the dapi one still work, I think they'll removed it.

Comment (carlosmiei) [reply ts/src/binance.ts:559]: @sc0Vu got it, yeah I think we can remove it

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**